common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -81,8 +81,8 @@
             return false;
         }
 
-        var now = new Date().getTime();
-        var lastContribution = new Date(value).getTime();
+        var now = new window.Date().getTime();
```

Comment:
We have to explicitly say we're initialising `new window.Date()` otherwise the compiler gives warning `SC_INEXISTENT_PROPERTY. Property getTime never defined on Date at applyRenderConditions.js line 84 : 35`, [which throws `RuntimeException`](https://github.com/guardian/frontend/blob/main/common/app/common/JsMinifier.scala#L65-L71):

```
java.lang.RuntimeException: JSC_INEXISTENT_PROPERTY. Property getTime never defined on Date<?> at applyRenderConditions.js line 84 : 35}
[info]   at common.JsMinifier$.compileUnsafe(JsMinifier.scala:66)
[info]   at common.JsMinifier$.$anonfun$unsafeCompileWithStandardOptimisation$1(JsMinifier.scala:76)
[info]   at scala.util.Try$.apply(Try.scala:210)
[info]   at common.JsMinifier$.unsafeCompileWithStandardOptimisation(JsMinifier.scala:76)
[info]   at common.InlineJs$.optimizeJs(JsMinifier.scala:108)
```